<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reserveDialog</class>
 <widget class="QDialog" name="reserveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>397</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>预定房间</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/bookin.png</normaloff>:/icons/icons/bookin.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>您的信息</string>
   </property>
   <widget class="QLineEdit" name="lineEditPhoneNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>联系方式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>30</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>房间信息</string>
   </property>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>房间类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxType">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>标准间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>双人间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大床间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商务间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VIP贵宾间</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>入住时间：</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditCheckIn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRoomNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>预定房间数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>入住天数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDayNum">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>预定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBKG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelBKG</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>okButton</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditPhoneNum</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>spinBoxRoomNum</tabstop>
  <tabstop>dateEditCheckIn</tabstop>
  <tabstop>spinBoxDayNum</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
